Atmosphere/RaymarchedClouds/DeferredRaymarchedVolumetricCloudsRenderer.cs

```diff
@@ -4,10 +4,32 @@
 using UnityEngine.Rendering;
 using ShaderLoader;
 using Utils;
-
+using UnityEngine.XR;
 
 namespace Atmosphere
 {
+    struct FlipFlop<T>
```

Comment:
I'd appreciate if you could move this and the related create/release methods + the camera matrices methods to a separate file, maybe VRUtils or something like that.